### PR TITLE
yii\grid add closure object content options from \yii\data\Column

### PR DESCRIPTION
**Pull request by sanchezzzhak**
| Q             | A
| ------------- | ---
| Is bugfix?    | not sure
| New feature?  | no
| Tests pass?   | yes

### CODE REVIEW

File: framework/grid/Column.php [hunk 1]
```diff
@@ -102,7 +102,7 @@ public function renderFooterCell()
      */
     public function renderDataCell($model, $key, $index)
     {
-        if ($this->contentOptions instanceof Closure) {
+        if ($this->contentOptions instanceof Closure|| is_callable($this->contentOptions)) {
```
Comment (swods) [inline framework/grid/Column.php]: Closure|| => Closure ||
&nbsp

**Comment by sanchezzzhak** [comment]
Can I restart the test?

**Comment by samdark** [comment]
Done.

**Comment by samdark** [comment]
OK. Tests pass.

Why this change was made? What's use case for it?

**Pull request closed without merging**